internal/schemautil/userconfig/handle.go

```diff
@@ -161,13 +170,19 @@ func handleArrayProperty(n string, p map[string]interface{}, t string) (map[stri
 				}
 			}
 
-			e, err = handleArrayOfAggregateTypeProperty(ipa)
+			req := map[string]struct{}{}
+
+			if sreq, ok := ia["required"].([]interface{}); ok {
+				req = SliceToKeyedMap(sreq)
+			}
+
+			e, err = handleArrayOfAggregateTypeProperty(ipa, req)
```

Comment:
Please translate: what are iof, ipa, ia?

Reply:
this is planned to be changed in future, after we release v4, but here are the abbreviations used:

`iof` is item one of, `ipa` is item properties asserted, `ia` is item asserted